src/main/java/docSharing/controller/AbstractController.java

```diff
@@ -44,66 +58,117 @@ public ResponseEntity<List<FileRes>> getAll(Long parentFolderId, Long userId) {
                 documents = documentService.getAllWhereParentFolderIsNull(userId);
 
             } catch (AccountNotFoundException e) {
+                logger.error("in AbstractController -> getAll -> " + e.getMessage());
                 throw new RuntimeException(e);
             }
         }
         return ResponseEntity.ok().body(convertToFileRes(folders, documents));
     }
 
+    /**
+     * convertToFileRes is an inner function of getAll, that gets a list of folders & documents,
+     * and return a list of FileRes entity which has the name,id and the type of a given file
+     * for the convenient of the client side which need
+     * to show all the files to user.
+     *
+     * @param folders   - List<Folder> to present.
+     * @param documents - List<Document> to present
+     * @return - List<FileRes>
+     */
     private List<FileRes> convertToFileRes(List<Folder> folders, List<Document> documents) {
+        logger.info("in AbstractController -> convertToFileRes");
         List<FileRes> fileResList = new ArrayList<>();
         for (Folder folder :
                 folders) {
 
             fileResList.add(new FileRes(folder.getName(), folder.getId(), Type.FOLDER, Permission.ADMIN, folder.getUser().getEmail()));
         }
-        for (Document document :
-                documents) {
+        for (Document document : documents) {
             fileResList.add(new FileRes(document.getName(), document.getId(), Type.DOCUMENT, Permission.ADMIN, document.getUser().getEmail()));
         }
         return fileResList;
     }
 
-
+    /**
+     * create is a request from the client to create a new file in a specific folder location.
+     *
+     * @param item - item of kind folder or document.
+     * @param c    - the class of the item, need to know to what service sends the request.
+     * @return - ResponseEntity.
+     */
     public ResponseEntity<String> create(GeneralItem item, Class c) {
         // make sure we got all the data from the client
+        logger.info("in AbstractController -> create");
```

Comment:
Try to make the logs more informative:
AbstractController - create: Trying to create an item of class ... 
When we read the logs from the file, it will be easier to understand the flow.